src/quic/node_quic_socket.h

```diff
@@ -166,7 +166,8 @@ class QuicPacket : public MemoryRetainer {
   SET_SELF_SIZE(QuicPacket);
 
  private:
-  std::vector<uint8_t> data_;
+  uint8_t data_[NGTCP2_MAX_PKTLEN_IPV4];
```

Comment:
Bringing same [comment](https://github.com/nodejs/quic/pull/388#discussion_r433141358) here.
Why do we use `NGTCP2_MAX_PKTLEN_IPV4` instead of something like `std::max(NGTCP2_MAX_PKTLEN_IPV4, NGTCP2_MAX_PKTLEN_IPV6)`? will just ipv4 len be fine?

Reply:
The difference between the two is minimal (a handful of bytes) and `NGTCP2_MAX_PKTLEN_IPV4` is the larger of the two.

Reply:
Then it would be a nit: I'd prefer to have this as i.e. define/constexpr of `std::max` to avoid possibly missing this in a future ngtcp update and to be generally more correct.

Reply:
We can do that but, just to be clear, the sizes here are dictated by the QUIC specification and not by ngtcp2.